### PR TITLE
Add Leaky relu activation function

### PR DESCRIPTION
**Pull request by Shashi456**
I was going to add SELU as well, but it has a prerequisite of initializing with variance scaling, which needs flow control. I'll add that later. Test and build passes locally.

Working on Separable layers next and the layer abstraction we spoke about.

### CODE REVIEW

**Comment by Shashi456** [comment]
Also are there any other activations to be added?

File: Sources/TensorFlow/Operators/Math.swift [hunk 1]
```diff
@@ -1014,6 +1014,23 @@ func _vjpElu<T: TensorFlowFloatingPoint>(
     return (y, { v in Raw.eluGrad(gradients: v, outputs: y) })
 }
 
+/// Computes `leakyRelu` of the specified tensor element-wise.
```
Comment (rxwei) [inline Sources/TensorFlow/Operators/Math.swift]: - The API documentation here is a bit redundant in that it does not explain what `leakyRelu` is. Other documentation like `relu` is also unclear. It should be changed to using the standard terminology.
- Could you please make sure #160 is addressed in future PRs?

```suggestion
/// Returns a tensor by applying the leaky ReLU activation function to the specified tensor element-wise.
```

File: Sources/TensorFlow/Operators/Math.swift [hunk 2]
```diff
@@ -1014,6 +1014,23 @@ func _vjpElu<T: TensorFlowFloatingPoint>(
...
+
+@inlinable
+func _vjpLeakyRelu<T: TensorFlowFloatingPoint>(
+    _ x: Tensor<T>,
+    alpha: Double = 0.2
+) -> (Tensor<T>, (Tensor<T>) -> Tensor<T>) {
+    return (leakyRelu(x, alpha: alpha), { v in Raw.leakyReluGrad(gradients: v, features: x, alpha: alpha) })
+}
+
+
```
Comment (rxwei) [inline Sources/TensorFlow/Operators/Math.swift]: Remove extra line.

File: Sources/TensorFlow/Operators/Math.swift [hunk 3]
```diff
@@ -1014,6 +1014,23 @@ func _vjpElu<T: TensorFlowFloatingPoint>(
...
+@inlinable
+@differentiable(wrt: x, vjp: _vjpLeakyRelu)
+public func leakyRelu<T: TensorFlowFloatingPoint>(_ x: Tensor<T>, alpha: Double = 0.2) -> Tensor<T> {
+    Raw.leakyRelu(features: x, alpha: alpha)
+}
+
+@inlinable
+func _vjpLeakyRelu<T: TensorFlowFloatingPoint>(
+    _ x: Tensor<T>,
+    alpha: Double = 0.2
```
Comment (rxwei) [inline Sources/TensorFlow/Operators/Math.swift]: Please remove default values from VJP functions.

```suggestion
    alpha: Double
```

File: Sources/TensorFlow/Operators/Math.swift [hunk 4]
```diff
@@ -1014,6 +1014,23 @@ func _vjpElu<T: TensorFlowFloatingPoint>(
...
+) -> (Tensor<T>, (Tensor<T>) -> Tensor<T>) {
+    return (leakyRelu(x, alpha: alpha), { v in Raw.leakyReluGrad(gradients: v, features: x, alpha: alpha) })
```
Comment (rxwei) [inline Sources/TensorFlow/Operators/Math.swift]: Please make sure all lines fit within 100 columns.

File: Sources/TensorFlow/Operators/Math.swift [hunk 5]
```diff
@@ -1014,6 +1014,23 @@ func _vjpElu<T: TensorFlowFloatingPoint>(
     return (y, { v in Raw.eluGrad(gradients: v, outputs: y) })
 }
 
+/// Computes `leakyRelu` of the specified tensor element-wise.
+/// Specifically, computes `max(features, features * alpha)`.
+@inlinable
+@differentiable(wrt: x, vjp: _vjpLeakyRelu)
+public func leakyRelu<T: TensorFlowFloatingPoint>(_ x: Tensor<T>, alpha: Double = 0.2) -> Tensor<T> {
```
Comment (rxwei) [inline Sources/TensorFlow/Operators/Math.swift]: - For consistency with tf.keras, `alpha` should be `0.3`.
- Fit within 100 columns.

Comment (Shashi456) [reply Sources/TensorFlow/Operators/Math.swift]: @rxwei so 0.2 is the default in the raw c++ apis tho, what would you like to choose?

File: Sources/TensorFlow/Operators/Math.swift (hunk 1 above)
Comment (Shashi456) [reply Sources/TensorFlow/Operators/Math.swift]: @rxwei I'll start working on #160 for all the files, I just take the neighborhood functions as references and the documentation is pretty inconsistent so I make these mistakes. I'll work on #160 for math.swift this week.

File: Sources/TensorFlow/Operators/Math.swift (hunk 5 above)
Comment (rxwei) [reply Sources/TensorFlow/Operators/Math.swift]: Ok, in that case, `0.2` is good. Thanks!

File: Sources/TensorFlow/Operators/Math.swift [hunk 6]
```diff
@@ -1014,6 +1014,27 @@ func _vjpElu<T: TensorFlowFloatingPoint>(
     return (y, { v in Raw.eluGrad(gradients: v, outputs: y) })
 }
 
+/// Returns a tensor by applying the leaky ReLU activation function to the specified tensor element-wise.
+/// Specifically, computes `max(features, features * alpha)`.
```
Comment (rxwei) [inline Sources/TensorFlow/Operators/Math.swift]: `features` is undefined.

```suggestion
/// Specifically, computes `max(x, x * alpha)`.
```

File: Sources/TensorFlow/Operators/Math.swift [hunk 7]
```diff
@@ -1014,6 +1014,27 @@ func _vjpElu<T: TensorFlowFloatingPoint>(
     return (y, { v in Raw.eluGrad(gradients: v, outputs: y) })
 }
 
+/// Returns a tensor by applying the leaky ReLU activation function to the specified tensor element-wise.
```
Comment (rxwei) [inline Sources/TensorFlow/Operators/Math.swift]: Fit within 100 columns.

**Pull request merged**